lib/streamlit/elements/lib/built_in_chart_utils.py

```diff
@@ -912,11 +914,13 @@ def _get_color_encoding(
             if len(color_values) != len(y_column_list):
                 raise StreamlitColorLengthError(color_values, y_column_list)
 
-            if len(color_value) == 1:
+            if len(color_values) == 1:
```

Comment:
@LukasMasuch can you sanity check that the previous check was wrong and we have to use this variable here? mypy reported "unreachable code" here

Reply:
hmm, I think both versions are correct since also `color_value` is a list or tuple. Not sure why mypy complained